modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/disk/TrackableFileIoFactory.java

```diff
@@ -149,7 +149,23 @@ private void trackWrites(int written) {
                 metrics.trackOffloadingWritten(written);
 
             if (tracker != null)
-                tracker.addTotalWrittenOnDisk(written);
+                tracker.spill(written);
+        }
+
+        /** {@inheritDoc} */
+        @Override public void clear() throws IOException {
+            super.clear();
+
+            tracker.unspill(tracker.writtenOnDisk());
+        }
+
+        /** {@inheritDoc} */
+        @Override public void close() throws IOException {
+            super.close();
+
+            tracker.unspill(tracker.writtenOnDisk());
```

Comment:
Why do we count `unspill` bytes here, but not in `trackReads()`?

Reply:
because when you read some portion of data it still remains on disk.

Reply:
Ok, I think we need to come to the same understanding of the **unspilling** term. I've found the only one [definition](https://digitalcommons.wpi.edu/cgi/viewcontent.cgi?article=1638&context=etd-theses) of this word:

> We call the process of flushing a portion
of the operator’s state to disk spilling and we call the process of bringing back to main
memory data spilled to disk unspilling. 

I found your definition in the code:

>  Amount of bytes deleted from disk.

I'm not sure which one is more correct. What do you think?